core/src/main/resources/messages/messages_en.yml

```diff
@@ -489,6 +489,55 @@ Particle-Effects:
   FireWaves:
     name: '&e&lFire Waves'
     Description: '&7&oSome say these waves of fire act as a shield...'
+  CursedHalo:
+    name: '&5&lCursed Halo'
+    Description: |-
+      &7&oIt brings great misfortune to
+      &7&oanyone who dares to wear it.
+  VolcanicHalo:
+    name: '&c&lVolcanic Halo'
+    Description: |-
+      &7&oA halo composed of
+      &7&oeternal flames.
+  BalefulFootprints:
+    name: '&5&lBaleful Footprints'
```

Comment:
I wonder if a synonym of baleful would be better, like menacing? (I had to google what 'baleful' means 😄 )

Reply:
Fixed. Renamed to "Cursed Footprints".